geowrangler/data_download.py

```diff
@@ -0,0 +1,48 @@
+# AUTOGENERATED! DO NOT EDIT! File to edit: notebooks/05_data_download.ipynb (unless otherwise specified).
```

Comment:
Hi @jtmiclat maybe we should rename the module to `datasets.geofabrik` since the code is specific to downloading the geofabrik  data. 

This would make it  easier to add other datasets in the future like `datasets.ookla` without going back and modifying this module.

Reply:
@butchtm Alright will do